### DISCUSSION TITLE
Allow cheats in warmup for practise purposes

### DISCUSSION THREAD

**Discussion by mittermichal** [etlegacy/etlegacy#1898] [Ideas]
Allow cheats like `noclip`, `god` and `give` in warmup for players to be able to practice before game.
Cheats that could also be added:
- Save/load position and no-collision could be used for practicing jumps
- display surfaces where landmine can be planted
- display volume where dynamites can be planted
- show missile camera to practice tricky grenades and riflenades - ported here: https://github.com/mittermichal/etlegacy/tree/riflecam

Should be controlled by server cvar and enabled in competition configs by default.

**Comment by keithmss** [comment]
Is there a CVAR for tracing grenade trajectory ? This could be useful during warmup as well. If that doesn’t exist right now maybe that could be good to implement as a separate feature.

**Reply by mittermichal** [reply]
Grenade trajectory is shown for shoutcaster or when cheats are enabled, but they aren't useful for riflenade shots through whole map
